src/lineable.js

```diff
@@ -80,7 +80,7 @@
 
             this.set.clear();
 
-            for (var i = 0; i < array.length; ++i) {
+            for (var i = 0; i < array.length - 1; ++i) {
```

Comment:
Last element is current position, right? We no need draw circle for it

Reply:
Thats not entirely clear to me. Why dont we need a circle for the current position? Imo the current position should how the representation of what is there, when you click and add the current position to the pointlist. Hence the point at the current position

Reply:
hm you are right. I guess we can merge it then :)

Reply:
I could fix it in this patch, but it already merged. 
And I did't make minified version.
When will you do the next release?

Reply:
uh thats what you mean. So there is a real bug in there. Jeah, it would be great if you could fix that one!

Reply:
I will be work on it in the near future :)

Reply:
cool, thank you!